calendar_public_holiday/hooks.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2024 Camptocamp
```

Comment:
Sorry for not noticing in my previous comment. As this is used in a hook, not on a migration script (that's what I meant for "business logic"), you should add the external dependency.